drupal_module_template.info.yml

```diff
@@ -0,0 +1,7 @@
+name: Drupal module template
+description: Drupal module template.
+package: Pronovix
+
+type: module
+core: 8.x
+php: "7.1"
```

Comment:
I thought we're on php-7.2+ now.

Reply:
Our module's minimum requirement is still 7.1, we just use PHP 7.2 as default in the testing matrix. If you read my previous comment carefully then you should see I did not say we bump the minimum required version by our modules - at least, not yet.